backend/src/v5/services/eventsListener/components/modelEvents.js

```diff
@@ -88,7 +88,7 @@ const revisionAdded = async ({ teamspace, project, model, revId, modelType, cali
 			{ _id: 0, tag: 1, author: 1, timestamp: 1, desc: 1, rFile: 1, format: 1, statusCode: 1, revCode: 1 });
```

Comment:
Background colour of the viewer should change as depicted by viewer settings
![image](https://github.com/user-attachments/assets/9be45dc1-3f1a-4a0d-983f-fe77e4f427ee)